tests/acceptance/features/bootstrap/WebDavLockingContext.php

```diff
@@ -127,7 +184,39 @@ public function lockFileUsingWebDavAPI(string $user, string $file, TableNode $pr
 	}
 
 	/**
-	 * @Given user :user has locked file/folder :file setting the following properties
+	 * @When user :user locks file :file inside the space :space using the WebDAV API setting the following properties
+	 *
+	 * @param string $user
+	 * @param string $file
+	 * @param string $space
+	 * @param TableNode $properties table with no heading with | property | value |
+	 *
+	 * @return void
+	 */
+	public function lockFileInProjectSpaceUsingWebDavAPI(string $user, string $file, string $space, TableNode $properties) {
+		$spaceId = $this->spacesContext->getSpaceIdByName($user, $space);
+		$fullUrl = $this->featureContext->getBaseUrl() . '/dav/spaces/' . $spaceId . '/' . $file;
+		$response = $this->lockFileInSpace($user, $fullUrl, $properties, false, false);
```

Comment:
can they be by default?
```suggestion
		$response = $this->lockFileInSpace($user, $fullUrl, $properties);
```